src/env/dev/terraform.tfvars

```diff
@@ -310,3 +314,20 @@ fdr_delete_retention_days       = 30
 reporting_fdr_function_kind     = "Linux"
 reporting_fdr_function_sku_tier = "Standard"
 reporting_fdr_function_sku_size = "S1"
+
+# gpd
+gpd_plan_kind     = "Linux"
+gpd_plan_sku_tier = "Standard"
+gpd_plan_sku_size = "S1"
+
+reporting_batch_function_autoscale_minimum = 1
+reporting_batch_function_autoscale_maximum = 3
+reporting_batch_function_autoscale_default = 1
```

Comment:
autoscale rule is a bit more complex with shared plan, probably u need to configure only 1 autoscale rule

Reply:
👍